client/src/HeaderPage.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
```

Comment:
Maybe just me but I got confused by this component name, since it has Page in the name I was thinking it was a page but it's just a header.

Reply:
This is a good point, I think Header would be better as the component name.